ct/wallos.sh

```diff
@@ -63,14 +63,19 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
   msg_info "Updating ${APP} to ${RELEASE}"
   cd /opt
   wget -q "https://github.com/ellite/Wallos/archive/refs/tags/v${RELEASE}.zip"
-  mv /opt/wallos/db/wallos.db /opt/wallos.db
-  mv /opt/wallos/images/uploads /opt/images/uploads
+  mkdir -p /opt/uploads
+  if [ -f "/opt/wallos/db/wallos.db" ]; then
+  mv "/opt/wallos/db/wallos.db" "/opt/wallos.db"
+  fi
+  mv /opt/wallos/images/uploads/* /opt/uploads_backup/
```

Comment:
Should be /opt/uploads

Reply:
For what? Its only a Backup for 4 lines of Code. Moved Back later

Reply:
Your mkdir creates a different folder than the one you're trying to use